muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/records/body/application/records.tsx

```diff
@@ -224,21 +224,21 @@ class Records extends React.Component<RecordsProps, RecordsState> {
                     }
                     return <ApplicationListItem className={`course course--studies ${extraClassNameState}`} key={workspace.id} onClick={this.goToWorkspace.bind(this, user, workspace)}>
                       <ApplicationListItemHeader modifiers="course" key={workspace.id}>
-                        <span className="text text--course-icon icon-books"></span>
-                        <span className="text text--list-item-title">{workspace.name} {workspace.nameExtension ? "(" + workspace.nameExtension + ")" : null}</span> 
+                        <span className="application-list__header-icon icon-books"></span>
+                        <span className="application-list__header-primary">{workspace.name} {workspace.nameExtension ? "(" + workspace.nameExtension + ")" : null}</span> 
                         {getEvaluationRequestIfAvailable(this.props, workspace)}
                         {getAssessments(this.props, workspace)}
                         {getActivity(this.props, workspace)}
                       </ApplicationListItemHeader>
                     </ApplicationListItem>
                   })}
                 {record.transferCredits.length ? 
-                  <div className="application-list__header"><h3 className="text text--studies-list-header">{this.props.i18n.text.get("plugin.records.transferCredits")} ({storedCurriculumIndex[record.groupCurriculumIdentifier]})</h3></div> : null}
+                  <div className="application-list__header-container"><h3 className="application-list__header">{this.props.i18n.text.get("plugin.records.transferCredits")} ({storedCurriculumIndex[record.groupCurriculumIdentifier]})</h3></div> : null}
```

Comment:
Shouldn't the h3 elements className be application-list__header-title as used previously?

If we have application-list__header-container element then we should have some child node as application-list__header-title where only child element is text node. At least previously className application-list__header-title has been set to an element where only child node is text node.